File: OPENAPI-REST-API/swagger-client/scala-akka-http-server/src/main/scala/io/swagger/server/model/BackupLoginResponse.scala
package io.swagger.server.model


/**
 * = BackupLoginResponse =
 *
 * Login session response for backup storage.
 *
 * @param success Indicates whether a login session was created.
 * @param text Login URL or error text returned by the storage provider.
 */
case class BackupLoginResponse (
  success: Option[Boolean],
  text: Option[String]
)

File: OPENAPI-REST-API/openapi-client/javascript-closure-angular/API/Client/BackupLoginResponse.js
goog.provide('API.Client.BackupLoginResponse');

/**
 * Login session response for backup storage.
 * @record
 */
API.Client.BackupLoginResponse = function() {}

/**
 * Indicates whether a login session was created.
 * @type {!boolean}
 * @export
 */
API.Client.BackupLoginResponse.prototype.success;

/**
 * Login URL or error text returned by the storage provider.
 * @type {!string}
 * @export
 */
API.Client.BackupLoginResponse.prototype.text;

